src/main/resources/templates/authorize.html

```diff
@@ -5,6 +5,16 @@
     <title>Authorize iworkflows</title>
 </head>
 <body>
+<div>
+    Logged in user: <b th:inline="text" class="user"> [[${#httpServletRequest.remoteUser}]] </b>
+    <div>
+        <br>
+        <form th:action="@{/logout}" method="POST">
+            <input type="submit" value="Logout"/>
+        </form>
+        <br>
+    </div>
+</div>
```

Comment:
since this is repeating, better to add as a fragment, anyway it's ok for now.

Reply:
Okay, I 'll address that in a separate PR

Reply:
thanks